pbxplore/analysis/visualization.py

```
@@ -21,7 +21,7 @@
     # it is 'SymbolColor'. Here, we change to always have 'ColorGroup'.
     try:
         ColorGroup = weblogolib.SymbolColor
-    except NameError:
+    except:
```

Does it really have to be a bare except?